Sources/ArgumentParser/Parsable Properties/Option.swift

```diff
@@ -294,18 +299,22 @@ extension Option where Value: ExpressibleByArgument {
       completion: completion)
   }
 
-  /// Creates a property with no default value.
+  /// Creates a required property that reads its value from a labeled option.
+  ///
+  /// This initializer is used when you declare an `@Option`-attributed property
+  /// that has an `ExpressibleByArgument` type, but without a default value:
   ///
-  /// This method is called to initialize an `Option` without a default value such as:
   /// ```swift
-  /// @Option var foo: String
+  /// @Option var name: String
   /// ```
   ///
   /// - Parameters:
-  ///   - name: A specification for what names are allowed for this flag.
-  ///   - parsingStrategy: The behavior to use when looking for this option's value.
+  ///   - name: A specification for what names are allowed for this option.
```

Comment:
The rename from `foo` to `name` may add to confusion, since there is also a `name` argument in the initializer.